packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -245,6 +259,21 @@ class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProvider
     widget.onExpansionChanged?.call(_isExpanded);
   }
 
+  ListTileControlAffinity effectiveAffinity(ListTileControlAffinity affinity) {
```

Comment:
For the sake of consistency make this private:  _effectiveAffinity(). Is this really worth factoring into a separate method?

Reply:
Thanks for pointing out that it was accidentally made public. The method is used in two places now.